nrfcloud/validatedFetch.ts

```diff
@@ -44,12 +46,14 @@ export const validatedFetch =
 	async <Schema extends TObject>(
 		{ resource }: { resource: string },
 		schema: Schema,
+		init?: RequestInit,
```

Comment:
Unrelated change.

Reply:
This change is necessary due to the default behavior uses `GET` method and `Content-Type` as `application/json`

Reply:
Changed in the refactor.